lib/routes/notifications-participant.js

```diff
@@ -0,0 +1,158 @@
+const router = require('koa-router')()
+const models = require('../models')
+
+router
+  .get('/notifications/participant/:fbid', async (ctx) => {
+    let fbid = ctx.params.fbid
+    await models.db.participant.findOne({
+      where: {fbid}
+    }).then(async (participant) => {
+      if (participant) {
+        console.log('participant found for fbid=' + fbid)
+        ctx.body = await models.db.participant_notification.findAll({
+          where: {participant_id: participant.id}
```

Comment:
Instead of doing this nested fetch on participant_notification, just add an include to the `models.db.participant.findOne({where: {fbid}})` call like so:
```
ctx.body = await models.db.participant.findOne({
  where: {fbid},
  include: models.db.notification
})
```

Reply:
Note that the include can also be a dictionary that has a where clause which operates on the notification, so you can do the eventId-based filtering from within that query also.